app/controllers/places_controller.rb

```diff
@@ -53,9 +53,12 @@ def show
 
 protected
 
-  def error_400(error)
-    error_message = error.message.gsub("MapitApi::", "").camelize(:lower)
-    render status: :bad_request, json: { error: error_message.to_s }
+  def error_400_invalid_postcode
+    render status: :bad_request, json: { error: "invalidPostcodeError" }
```

Comment:
Minor: can we turn the error message into PascalCase or an actual error message to compare against in Frontend. As we now interpret the message in two Ruby processes (rather than building from a Mapit class), it seems strange to use a non Ruby case.